practices/practice-one/src/components/Header/Header.tsx

```diff
@@ -7,18 +7,52 @@ import Paragraph from "components/Paragraph/Paragraph";
 import Input from "components/Input/Input";
 import { navLinks } from "constants/navLinks";
 import Form from "components/Form/Form";
-
 import "./header.css";
-import { FormEvent, useState, MouseEvent } from "react";
+import { FormEvent, useState, MouseEvent, ChangeEvent } from "react";
 import Overlay from "components/Overlay/Overlay";
+import useStore from "hooks/useStore";
+import { ContextAction } from "constants/contextAction";
+import validation from "helpers/validation/empty";
+
 function Header() {
+  const [state, dispath] = useStore();
   const [popup, setPopup] = useState(false);
 
-  const handleSubmit = (event: FormEvent) => {};
+  const { blog } = state;
+
+  const handleSubmit = (event: FormEvent) => {
+    event.preventDefault();
+    const { dataFields, error } = validation(blog);
+    if (error) {
+      // I will add the error message later
```

Comment:
// TODO: I will add the error message later

Reply:
I will update